server/express-server.ts

```diff
@@ -203,6 +204,12 @@ app.put(CONFIG_TWITCH_LOG_ENDPOINT, (req, res) => {
   res.send(PersistenceInstance.updateTwitchLog(req.body.twitchLog));
 });
 
+app.put(CONFIG_TWITCH_BOT_ENDPOINT, (req, res) => {
+  // update config
+  const {bot, botName, botToken} = req.body.twitch;
+  res.send(PersistenceInstance.updateTwitchBot(bot, botName, botToken));
```

Comment:
if you just use the bot-object itself, you dont need add all other properties everytime (on the methods in app / server) ... if there is need to add another property I mean